cookiecutter/config.py

```diff
@@ -60,13 +60,8 @@ def get_config(config_path):
         )
 
     logger.debug('config_path is %s', config_path)
-    with open(config_path, encoding='utf-8') as file_handle:
-        try:
-            yaml_dict = poyo.parse_string(file_handle.read())
-        except poyo.exceptions.PoyoException as e:
-            raise InvalidConfiguration(
-                'Unable to parse YAML file {}. Error: {}'.format(config_path, e)
-            )
+    with open(config_path, 'r') as yaml_file:
+        yaml_dict = yaml.load(yaml_file, Loader=yaml.FullLoader)
```

Comment:
We should catch the `yaml.YAMLError` exception and raise a `InvalidConfiguration` exception in case of a malformed yaml file. 

Also, we should use the `yaml.SafeLoader` loader: we don't need the yaml features that it doesn't support (they weren't supported by poyo either).